erc20-load-test-tool/script/.env

```diff
@@ -1,4 +1,5 @@
 # SCRIPT ENV VARIABLES
+export DOWNLOAD_JAR=true
```

Comment:
can we add something like, if jar is not present in the target file then also download it from the artifactory?